### PR TITLE
Add 'Error loading page' case for validating correctness of test-perf data

### PR DESCRIPTION
**Pull request by asajeffrey**
Add case for `Error loading page` title to test-perf runner.

---

- [X] `./mach build -d` does not report any errors
- [X] `./mach test-tidy` does not report any errors
- [X] These changes do not require tests because this is test infrastructure

---
This change is [Reviewable](https://reviewable.io/reviews/servo/servo/19205)

### CODE REVIEW

File: etc/ci/performance/runner.py [hunk 1]
```diff
@@ -116,6 +116,7 @@ def valid_timing(timing, testcase=None):
         if (timing is None or
                 testcase is None or
                 timing.get('title') == 'Error response' or
```
Comment (jdm) [inline etc/ci/performance/runner.py]: I think we can get rid of this case; I don't see any code that would generate a page with that title.

**Review by jdm — CHANGES_REQUESTED** [review]

**Comment by edunham** [comment]
r=me w/ jdm's changes, this looks good

File: etc/ci/performance/runner.py (hunk 1 above)
Comment (asajeffrey) [reply etc/ci/performance/runner.py]: Done.

**Comment by jdm** [comment]
@bors-servo: retry
* #19234

**Pull request merged**